synthnn/performance/cuda_backend.py

```
@@ -17,6 +17,11 @@ def __init__(self, device_id: int = 0):
         super().__init__(device_id)
         self.cupy_available = False
         self.torch_cuda_available = False
+
+    @property
+    def backend_type(self):
```

Ruff (0.11.9)

Missing return type annotation for public function `backend_type`

(ANN201)